install-php-extensions

```diff
@@ -181,9 +181,19 @@ buildRequiredPackageLists () {
 	buildRequiredPackageLists_volatile=''
 	buildRequiredPackageLists_distro="$(getDistro)"
 	buildRequiredPackageLists_phpv=$1
-	if test "$buildRequiredPackageLists_distro" = 'alpine'; then
-		buildRequiredPackageLists_volatile="$PHPIZE_DEPS"
-	fi
+	case "$buildRequiredPackageLists_distro" in
+		alpine)
+			buildRequiredPackageLists_volatile="$PHPIZE_DEPS"
+			;;
+		debian)
+			if test -n "$(apt-cache search libssl1.0 | grep -E ^libssl1\.0)"; then
+				# Debian 9, uses libssl1.0 instead of libssl, due to conflict with libssh-dev
```

Comment:
`Debian less than 10` since jessie is included, I forgot about jessie when writing this comment

Reply:
:( Sorry, just tested with jessie, this does not work on jessie

`E: Couldn't find any package by regex 'libssl1.0-dev'`